frontend-project/src/models/users.ts

```diff
@@ -0,0 +1,46 @@
+import { fetchAll } from '@/services/users';
+import { Effect, EffectsCommandMap } from 'dva';
+import { Reducer, AnyAction } from 'redux';
+
+export interface User {
+  username: string;
+  email: string;
+  firstName: string;
+  lastName: string;
+  id: number;
+}
+
+export interface UsersState {
+  users: User[];
```

Comment:
Czy to oznaca @kuskoman że efektywnie idziemy z tym co zapropnowałeś w #298?

Reply:
w sumie to tak, chyba, że masz inną propozycję